src/ratelimit_action.rs

```diff
@@ -19,6 +20,8 @@ struct DescriptorEntryBuilder {
     pub expression: Expression,
 }
 
+pub const KNOWN_ATTRIBUTES: [&str; 2] = ["ratelimit.domain", "ratelimit.hits_addend"];
```

Comment:
nit: no need for this to be `pub`lic afaict

Reply:
done